odl/tomo/backends/skimage_radon.py

```diff
@@ -31,23 +33,24 @@ def skimage_proj_space(geometry, volume_space, proj_space):
     det_part = uniform_partition(-det_width / 2, det_width / 2, padded_size)
 
     part = geometry.motion_partition.insert(1, det_part)
-    space = uniform_discr_frompartition(
-        part, interp=proj_space.interp, dtype=proj_space.dtype
-    )
+    space = uniform_discr_frompartition(part, dtype=proj_space.dtype)
     return space
 
 
-def clamped_interpolation(skimage_proj_space, sinogram):
+def clamped_interpolation(skimage_range, sinogram):
     """Interpolate in a possibly smaller space.
 
     Clip all points to fit within the bounds of the given space.
     """
     min_x = skimage_proj_space.domain.min()[1]
     max_x = skimage_proj_space.domain.max()[1]
 
-    def interpolator(x):
+    def interpolator(x, out=None):
         x = (x[0], np.clip(x[1], min_x, max_x))
-        return sinogram.interpolation(x, bounds_check=False)
+        interpolator = linear_interpolator(
```

Comment:
I don't know why it was there in the first place, but why remove the `bounds_check=False`?

Reply:
I think I removed it for debugging purposes, but it should go back in. We construct `x` to be in bounds, so we can skip the (potentially expensive) check.

Reply:
Turns out that the created interpolator doesn't do bounds checking, so no need for `bounds_check=False`.